requirements/requirements-test.txt

```diff
@@ -5,10 +5,9 @@ pytest
 coverage==7.2.3
 git+https://github.com/hpcaitech/pytest-testmon
 torchvision
-transformers==4.33.0
```

Comment:
I'm wondering if we need to move some test dependencies into the infer dependencies, as currently the infer dependencies are all being installed from the requirements-test.txt file, which feels a bit strange.